plugins/wysiwygarea/plugin.js

```diff
@@ -716,3 +808,33 @@ CKEDITOR.config.disableNativeSpellChecker = true;
  * @param {CKEDITOR.editor} editor This editor instance.
  * @param {CKEDITOR.dom.element} data The element being added.
  */
+
+/**
+ * Document observation starting point. Native DOM object.
+ *
+ * To recreate editor `wysiwygarea` iframe after reattach editor to DOM:
+ *
+ * * make sure detachable element is placed any nested depth under observable element.
+ * * make sure editor is placed any nested depth under detachable element.
+ *
+ * See overview of hierarchy below (HTML pseudocode):
+ *
+ * ```
+ * <observable>
+ *   <detachable> <!-- Any nesting depth -->
+ *     <editor></editor> <!-- Any nesting depth -->
+ *   </detachable>
+ * </observable>
```

Comment:
Maybe adding `...` instead of the comment?

```suggestion
 * <observable>
 *    <...>
 *     <detachable>
 *       <...>
 *         <editor></editor>
 *       </...>
 *     </detachable>
 *    </...>
 * </observable>
```